utkdigitalinitiatives/UTKdrupal
# Trac-995 & Trac-935

**JIRA Ticket**: [TRAC-995](https://jira.lib.utk.edu/browse/TRAC-995) & [TRAC-935](https://jira.lib.utk.edu/browse/TRAC-935)

# What does this Pull Request do?

Merges in several changes to correct layout for datasets, search page and general theme changes. 

# What's new?
Correction in how the theme behaves on lower resolution screens and page layout behaves as expected. 

# How should this be tested?
Look at the readme examples of layout. Verify it looks as intended. Also do a few test submissions as a user and accept and publish as thesis_manager to verify no bugs were introduced. 

# Additional Notes:
n/a 

# Interested parties
Tag @utkdigitalinitiatives/digital-initiatives-developers
